Makefile

```diff
@@ -155,6 +155,7 @@ test: generate  ## Run the tests
 
 e2e-test: generate
 	@go test -race -coverpkg=./pkg/app -coverprofile=e2e-coverage.out ./pkg/app
```

Comment:
Can we move the test suite to e2e/ directory directly?

Reply:
I move these files to the app/, But I think it is ok for now but If we have more and more spec test may not be related those commands directly. we still need a independent location to put those specs.

Reply:
I mean move the tests in pkg/app/ to e2e/

Then all the e2e test cases are placed in the same dir.